### PR TITLE
Fix LfMerge config in base Docker image

### PR DESCRIPTION
**Pull request by rmunn**
This will allow LfMerge to find the Mongo database container. Future improvements might include looking for environment variables in LfMerge, but this is enough to get LfMerge working in the Docker environment.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/925)

### CODE REVIEW

**Review by rmunn — COMMENTED** [review]
Self-review for the sake of testing. I'm allowed to comment, but not approve: the Approve radio button is disabled if you're self-reviewing. Good.

image

**Review by megahirt — APPROVED** [review]
looks good - thanks Robin.

**Comment by rmunn** [comment]
Note: I used admin access to override the approvals, so that I wouldn't have to wait for the E2E tests to complete. This PR changes the base app image, and the new base app won't be pushed to Docker Hub until the PR is merged, which means that the E2E tests will actually not be testing any changes from this PR anyway. So I decided to save 20 minutes. It's been tested by hand on QA, so I feel okay about merging this.

**Pull request merged**
